shell/src/main/java/alluxio/shell/command/CheckConsistencyCommand.java

```diff
@@ -61,6 +61,14 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
     return 0;
   }
 
+  /**
+   * check consistency and find out inconsistent files.
```

Comment:
To keep the consistent style, the first word should be capitalized. It should be `Checks`.